### PR TITLE
Test profile

### PR DESCRIPTION
**Pull request by HengningZhang**
Initialized tests and documentation for the APIs relating to profile and password management.

### CODE REVIEW

File: application/profile.py [hunk 1]
```diff
@@ -18,33 +18,47 @@ def get_profile_dict():
...
+    """
     info = get_profile_dict()
-    return jsonify(results = info)
+    return jsonify(results = info),200
     
 @app.route("/api/change-password", methods=["POST"])
 def change_password():
+    """
+    change_password(old_password,new_password) -> Boolean
+    Change password for the current user.
```
Comment (LujieZ) [inline application/profile.py:46]: Return?

File: config.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 class BaseConfig(object):
     SECRET_KEY = "SO_SECURE"
     DEBUG = True
-    SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
+    SQLALCHEMY_DATABASE_URI = 'mysql://doadmin:rtrjvssd4s8bpp14@grickly-nyu-do-user-8731753-0.b.db.ondigitalocean.com:25060/test'
```
Comment (LujieZ) [inline config.py:9]: You shouldn't put the database key here. Use an env variable instead.

File: config.py [hunk 3]
```diff
@@ -15,6 +15,7 @@ class TestingConfig(object):
     TESTING = True
     DEBUG = True
     WTF_CSRF_ENABLED = False
-    SQLALCHEMY_DATABASE_URI = 'sqlite:///:memory:'
+    SQLALCHEMY_DATABASE_URI = 'mysql://doadmin:rtrjvssd4s8bpp14@grickly-nyu-do-user-8731753-0.b.db.ondigitalocean.com:25060/test'
```
Comment (LujieZ) [inline config.py:18]: Same as above.

**Review by LujieZ — CHANGES_REQUESTED** [review]

**Comment by KevinSirius** [comment]
LGTM

**Comment by gokejian** [comment]
LGTM!

**Review by LujieZ — APPROVED** [review]

**Pull request merged**